module_intros/diffraction.StaticStructureFactor.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    traj = gsd.hoomd.open("../data/LJsampletraj.gsd", "rb")
</pre><p>You may not need the <code>../</code>  because <code>data</code>  should be symlinked in this directory. If not, please remove.</p><p> </p><p>Additionally, this should be used as a context manager in the next cell:</p><pre class="ql-syntax" spellcheck="false">with gsd.hoomd.open("data/LJsampletraj.gsd", "rb") as traj:
    for trame in traj:
        # ...
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>